Sources/Purchasing/Offering.swift

```diff
@@ -109,6 +109,7 @@ import Foundation
         return package(identifier: key)
     }
 
+    // swiftlint:disable:next cyclomatic_complexity
```

Comment:
Hmm might be good to try to refactor this to avoid this issue... But NABD.

Reply:
Yeah I thought about it but this needs to be in `init` in order to make the type immutable so I couldn't think of how to extract it out. Ultimately it's just a factor of the number of `case`s and putting a limit to that is kind of arbitrary.